app/src/main/java/woowacourse/movie/activity/MovieReservationActivity.kt

```diff
@@ -0,0 +1,162 @@
+import woowacourse.movie.view.MovieViewData
+import woowacourse.movie.view.ReservationData
+import woowacourse.movie.view.mapper.ReservationDetailMapper.toView
+import woowacourse.movie.view.widget.Counter
+import woowacourse.movie.view.widget.DateSpinner
+import woowacourse.movie.view.widget.LocalFormattedDate
+import woowacourse.movie.view.widget.LocalFormattedTime
+import woowacourse.movie.view.widget.MovieController
+import woowacourse.movie.view.widget.MovieView
+import woowacourse.movie.view.widget.ReservationButton
+import woowacourse.movie.view.widget.SaveStateCounter
+import woowacourse.movie.view.widget.SaveStateSpinner
+import woowacourse.movie.view.widget.TimeSpinner
+import java.time.LocalDateTime
+
+class MovieReservationActivity : AppCompatActivity() {
+    private val counter: SaveStateCounter by lazy {
+        SaveStateCounter(
+            Counter(
+                findViewById(R.id.movie_reservation_people_count_minus),
+                findViewById(R.id.movie_reservation_people_count_plus),
+                findViewById(R.id.movie_reservation_people_count),
+                Count(INITIAL_COUNT),
+            ),
+            COUNTER_SAVE_STATE_KEY,
+        )
+    }
+
+    private val dateSpinner: DateSpinner by lazy {
+        DateSpinner(
+            SaveStateSpinner(
+                DATE_SPINNER_SAVE_STATE_KEY,
+                findViewById(R.id.movie_reservation_date_spinner),
+            )
+        )
+    }
+
+    private val timeSpinner: TimeSpinner by lazy {
+        TimeSpinner(
+            SaveStateSpinner(
+                TIME_SPINNER_SAVE_STATE_KEY,
+                findViewById(R.id.movie_reservation_time_spinner),
+            )
+        )
+    }
+
+    private val reservationButton: ReservationButton by lazy {
+        ReservationButton(
+            findViewById(R.id.movie_reservation_button)
+        )
+    }
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContentView(R.layout.activity_movie_reservation)
+        initMovieReservationView(savedInstanceState)
+    }
+
+    private fun initMovieReservationView(savedInstanceState: Bundle?) {
+        makeBackButton()
+        val movie = intent.extras?.getSerializable<MovieViewData>(MovieViewData.MOVIE_EXTRA_NAME)
+        if (movie != null) {
+            makeCounter(savedInstanceState)
+            makeSpinners(savedInstanceState, movie)
+            renderMovie(movie)
+            makeReservationButtonClickListener(movie)
+        }
+    }
+
+    private fun makeBackButton() {
+        supportActionBar?.setDisplayHomeAsUpEnabled(true)
+    }
+
+    private fun makeCounter(savedInstanceState: Bundle?) {
+        counter.applyToView()
+        counter.load(savedInstanceState)
+    }
+
+    private fun makeSpinners(savedInstanceState: Bundle?, movie: MovieViewData) {
+        dateSpinner.make(
+            savedInstanceState = savedInstanceState, movie = movie, timeSpinner = timeSpinner
```

Comment:
```suggestion
            savedInstanceState = savedInstanceState, 
            movie = movie, 
            timeSpinner = timeSpinner
```
가로로 길어지는 경우에는 콤마 단위로 개행하시면 한 눈에 보기 편해집니다.